modules/wsl-distro.nix

```diff
@@ -179,10 +179,7 @@ with lib; {
             # preserve $PATH from parent
             variables.PATH = [ "$PATH" ];
             extraInit = ''
-              export WSLPATH=$(echo "$PATH" | tr ':' '\0' | command grep -az "^${cfg.wslConf.automount.root}" | tr '\0' ':')
-              ${if cfg.interop.includePath then "" else ''
-                export PATH=$(echo "$PATH" | tr ':' '\0' | command grep -avz "^${cfg.wslConf.automount.root}" | tr '\0' ':')
-              ''}
+              eval $(${nativeUtils}/bin/split-path --automount-root="${cfg.wslConf.automount.root}" ${if cfg.interop.includePath then "--include-interop" else ""})
```

Comment:
```suggestion
              eval $(${nativeUtils}/bin/split-path --automount-root="${cfg.wslConf.automount.root}" ${lib.optionalString cfg.interop.includePath then "--include-interop"})
```

Reply:
The `then` needs to go as well when using `optionalString`